Image Feed/ImagesList/ImagesListViewController.swift

```diff
@@ -1,6 +1,8 @@
 import UIKit
 
-class ImagesListViewController: UIViewController {
+final class ImagesListViewController: UIViewController {
+    private let ShowSingleImageSegueIdentifier = "ShowSingleImage"
```

Comment:
Наверное в уроке осталось с CamelCase'ом, однако любые переменные и константы стоит называть в нижнем camelCase
```suggestion
    private let showSingleImageSegueIdentifier = "ShowSingleImage"
```